tests/src/test/scala/system/basic/WskRestBasicTests.scala

```diff
@@ -44,6 +44,9 @@ class WskRestBasicTests extends TestHelpers with WskTestHelpers with WskActorSys
 
   val defaultAction: Some[String] = Some(TestUtils.getTestActionFilename("hello.js"))
 
+  val requireApiKeyAnnotation =
+    Option(WhiskProperties.getProperty("whisk.feature.requireApiKeyAnnotation")).map(_.toBoolean).getOrElse(true)
```

Comment:
Better to add a method `WhiskProperties.getBoolean(key, defaultValue)` and use that

Reply:
Changed as proposed